TROUBLESHOOTING.md

```diff
@@ -8,7 +8,8 @@ This guide helps you diagnose and resolve common issues with the Azure MCP Serve
   - [Console window is empty when running Azure MCP Server](#console-window-is-empty-when-running-azure-mcp-server)
   - [Can I select what tools to load in the MCP server?](#can-i-select-what-tools-to-load-in-the-mcp-server)
   - [Why does VS Code only show a subset of tools available?](#why-does-vs-code-only-show-a-subset-of-tools-available)
-  - [Bring your own language model key](#bring-your-own-language-model-key)
```

Comment:
Why did we remove this entry?

Reply:
It's not a part of this section as far as markdown formatting is concerned